### ISSUE TITLE
Add 'accessibility' link in the footer of the Resource Portal 

### ISSUE THREAD

**Issue by WendyEBurns** [digital-york/ncelp#228]
Email 2 October: 

Frank - is it ok if you add an 'Accessibility' link to the footer of the NCELP Resource Portal? Perhaps bottom right before the Legal statement e.g. Accessibility | Legal statement @ University of York?

The link should go to this page: (https://ncelp.org/accessibility/)

**Comment by websnacks** [comment]
Missing `title` tag on the accessibility link.

**Comment by websnacks** [comment]
Done, waiting for code review and merge into development.
